packages/core/js-client/src/services/SingleModuleSrv.ts

```diff
@@ -32,24 +31,21 @@ export const defaultGuard = (peer: FluencePeer) => {
   return allowOnlyParticleOriginatedAt(peer.keyPair.getPeerId());
 };
 
-export class Srv implements SrvDef {
+export class Srv {
   private services: Set<string> = new Set();
 
   constructor(private peer: FluencePeer) {
     this.securityGuard_create = defaultGuard(this.peer);
     this.securityGuard_remove = defaultGuard(this.peer);
   }
 
-  securityGuard_create: SecurityGuard<"wasm_b64_content">;
+  securityGuard_create: SecurityGuard;
 
-  async create(
-    wasm_b64_content: string,
-    callParams: CallParams<"wasm_b64_content">,
-  ) {
+  async create(wasm_b64_content: string, callParams: ParticleContext) {
     if (!this.securityGuard_create(callParams)) {
       return {
         success: false,
-        error: "Security guard validation failed",
+        error: ["Security guard validation failed"],
```

Comment:
What is security guard? What should user do about this error?

Reply:
This guard wont allow you to register arbitrary marine service from remote particle.
I changed error message, thanks